addon/commands/besluit-plugin/insert-article.js

```diff
@@ -0,0 +1,63 @@
+import { v4 as uuid } from 'uuid';
+import { insertHtml } from '@lblod/ember-rdfa-editor/commands/insert-html-command';
+
+export default function InsertArticleCommand(
```

Comment:
This command can be converted to a prosemirror command, this way you can also include a check in this command to determine whether or not an article may be inserted,